spot_wrapper/wrapper.py

```diff
@@ -518,21 +370,15 @@ def __init__(
         self._password = password
         self._hostname = hostname
         self._robot_name = robot_name
+        self._rates = rates
+        self._callbacks = callbacks
```

Comment:
```suggestion
        self._rates = rates or {}
        self._callbacks = callbacks or {}
```

Handle the case where rates and callbacks are unset in the init args (i.e. None).

Reply:
Agreed, done in [392da89](https://github.com/bdaiinstitute/spot_wrapper/pull/6/commits/392da898d027fc716a17f1cd7ff487183cc192cb)